### DISCUSSION TITLE
Flex row ? Custom repeater content ?

### DISCUSSION THREAD

**Discussion by OctaYann** [PrestaSafe/prettyblocks#189] [Q&A]
Hello, is there anyone who achieve created a flex row block where you can put blocks in it ?
Maybe with a repeater ?

Thank you (you can speak french if needed)

**Comment by PrestaSafe** [comment]
Le champs de type `repeater` sont dans notre Todo. 
Nous travaillons également sur un SDK afin que les développeurs puisse creer des champs custom (Vue3) et les faire fonctionner dans PrettyBlocks. 
Nous améliorons également notre système d'iframe pour mieux communiquer en PrettyBlocks et le front. 
Cordialement

**Comment by OctaYann** [comment]
Rebonjour,

Je vous remercie, de notre coté nous avons acheté le template afin de gagner du temps sur l'intégration des sites que nous développons, cependant il manque encore certains éléments important dans l'éditeur.
A quelle temporalité pensez vous qu'un système de ligne et colonnes flex pourra être dispo ?

En vous remerciant par avance
Et encore merci pour le travail founi

**Comment by OctaYann** [comment]
Bonjour,

J'ai vu que dans les pro blocks on a des champs repeatable mais est ce qu'on peut faire en sorte que ces champs soient polymophiques pour pouvoir construire un système de row et col ? Est ce qu'il y a un exemple quelque part ? @PrestaSafe 

Merci par avance.
